### DISCUSSION TITLE
How to change the default settings table to something else ? Can this be turned off ?

### DISCUSSION THREAD

**Discussion by sohaib13k** [codeigniter4/shield#609] [Q&A] [answered]
I'm using clients database, thus it doesn't contain settings table. Can table name be modified ? and also is it possible to turn this off and fetch values from .env or /config ?

**Comment by lonnieezell** [comment, marked as answer]
Shield uses [CodeIgniter/Settings](https://github.com/codeigniter4/settings) under the hood. There's no way to turn that off, but you can change the table used. 

Copy [this file](https://github.com/codeigniter4/settings/blob/develop/src/Config/Settings.php) into the `app/Config` folder and modify the settings as needed. You might need to modify the migration also.
